src/push-notifications/utils.ts

```diff
@@ -0,0 +1,9 @@
+import {NotificationConfigValue} from '@atb/push-notifications/types';
+import {isDefined} from '@atb/utils/presence';
+
+export function isConfigEnabled<T extends NotificationConfigValue>(
```

Comment:
Could consider renaming to `isNotificationConfigEnabled`

Reply:
I'm willing to consider your other more explicit naming suggestions, but I think this one reads just as well as it is. After all, it's just a helper that's not used outside the notifications feature.

Reply:
Fair point!